QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java

```diff
@@ -51,6 +53,16 @@ protected void collectTote() throws AutonomousModeOverException,
 		waitUntil(BooleanMixing
 				.invert((BooleanInputPoll) QuasarHelios.autoLoader));
 	}
+	
+	protected void setClamp(boolean value) throws InterruptedException, AutonomousModeOverException {
+		QuasarHelios.clamp.openControl.set(value);
+		waitForTime(30l);
+	}
```

Comment:
Also, is that `30l`? Specifying the long part makes it not clear whether it's `301` or `30l`.